nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/src/main/java/org/apache/nifi/web/security/oidc/StandardOidcIdentityProvider.java

```diff
@@ -439,8 +439,21 @@ private LoginAuthenticationToken convertOIDCTokenToLoginAuthenticationToken(OIDC
                 identity = claimsSet.getStringClaim(EMAIL_CLAIM);
                 logger.info("The 'email' claim was present. Using that claim to avoid extra remote call");
             } else {
-                identity = retrieveIdentityFromUserInfoEndpoint(oidcTokens);
-                logger.info("Retrieved identity from UserInfo endpoint");
+                final List<String> fallbackClaims = properties.getOidcFallbackClaimsIdentifyingUser();
+                if (fallbackClaims.size() > 0) {
+                    logger.info("fallbackClaims.size() : " + fallbackClaims.size());
```

Comment:
This is not necessary to log at the info level, and since it is a configuration property, it seems unnecessary to log even at the debug level.  Recommend at least changing to debug and using parameterized message strings or removing altogether.